src/navigation/UserVocabularyStackNavigator.tsx

```diff
@@ -24,6 +26,16 @@ const UserVocabularyStackNavigator = (): JSX.Element | null => {
         component={UserVocabularyOverviewScreen}
         options={({ navigation }) => options(overview, navigation)}
       />
+      <Stack.Screen
+        name='UserVocabularyList'
+        component={UserVocabularyListScreen}
+        options={({ navigation }) => options(overview, navigation)}
```

Comment:
I was also wondering, but in the designs wording is "overview", changed to "back" so it is always valid. If we want a more complex handling, I think we should open a new ticket, as it would also affect other screens (e.g. dictionary)